uuid_test.go

```diff
@@ -382,6 +395,19 @@ func TestScanUnsupported(t *testing.T) {
 	}
 }
 
+func TestScanNil(t *testing.T) {
+	u := UUID{}
```

Comment:
This value is already the expected result. This test would pass even if `Scan` never changes `u`.

Initialize this with a value, e.g. `u := UUID{0x6b, 0xa7, 0xb8, 0x10, 0x9d, 0xad, 0x11, 0xd1, 0x80, 0xb4, 0x00, 0xc0, 0x4f, 0xd4, 0x30, 0xc8}`, and we will be asserting that `Scan` changed `u` to `Nil`.